docs/collections/contribute.md

```diff
@@ -61,3 +61,22 @@ Use the [Prefect Collection template](https://github.com/PrefectHQ/prefect-colle
 ### List a project in the Collections Catalog
 
 To list your collection in the Prefect Collections Catalog, submit a PR to the Prefect repository adding a file to the `docs/collections/catalog` directory with details about your collection. Please use `TEMPLATE.yaml` in that folder as a guide.
+
+## Contributing features or bug fixes to Collections
```

Comment:
Maybe "Contribute fixes or enhancements to Collections"? I think that would be similar to the language we use elsewhere, such as release notes.